agent/fw_drupal8.c

```diff
@@ -605,6 +605,94 @@ NR_PHP_WRAPPER(nr_drupal94_invoke_all_with_clean) {
 NR_PHP_WRAPPER_END
 #endif  // OAPI
 
+static bool nr_validate_key_val_arr(nr_php_string_hash_key_t* key, zval* val) {
```

Comment:
The name made me think valid values would return true, and invalid values would return false.

Reply:
I agree with @zsistla's observation.

Reply:
fda25fe30b931b180adb4472eb8363aab24e47fb

Reply:
I think you meant 977e9efcdc1f42e42c346276778942b819b8523f, which looks good to me.